app/views/members/dues/_form.html.haml

```diff
@@ -8,7 +8,7 @@
     "$25.00 USD Monthly" => "medium_monthly",
     "$20.00 USD Monthly" => "20_monthly",
     "$15.00 USD Monthly" => "15_monthly",
-    "$10.00 USD Monthly" => "small_monthly" }, @current_plan || "30_monthly")
+    "$10.00 USD Monthly" => "small_monthly" }, @current_plan || "50_monthly")
```

Comment:
I think this needs to be `large_monthly` instead of `50_monthly` as that’s what the $50 plan in stripe is called